README.md

```diff
@@ -1,2 +1,234 @@
-# js-ipid
-The IPID DID method implementation in JavaScript
+# did-ipid
+
+[![NPM version][npm-image]][npm-url] [![Downloads][downloads-image]][npm-url] [![Build Status][travis-image]][travis-url] [![Coverage Status][codecov-image]][codecov-url] [![Dependency status][david-dm-image]][david-dm-url] [![Dev Dependency status][david-dm-dev-image]][david-dm-dev-url]
+
+[npm-url]:https://npmjs.org/package/did-ipid
+[downloads-image]:http://img.shields.io/npm/dm/did-ipid.svg
+[npm-image]:http://img.shields.io/npm/v/did-ipid.svg
+[travis-url]:https://travis-ci.org/ipfs-shipyard/js-ipid
+[travis-image]:http://img.shields.io/travis/ipfs-shipyard/js-ipid/master.svg
+[codecov-url]:https://codecov.io/gh/ipfs-shipyard/js-ipid
+[codecov-image]:https://img.shields.io/codecov/c/github/ipfs-shipyard/js-ipid/master.svg
+[david-dm-url]:https://david-dm.org/ipfs-shipyard/js-ipid
+[david-dm-image]:https://img.shields.io/david/ipfs-shipyard/js-ipid.svg
+[david-dm-dev-url]:https://david-dm.org/ipfs-shipyard/js-ipid?type=dev
+[david-dm-dev-image]:https://img.shields.io/david/dev/ipfs-shipyard/js-ipid.svg
+
+The [IPID DID method](https://github.com/jonnycrunch/ipid) implementation in JavaScript
+
+
+## Installation
+
+```sh
+$ npm install did-ipid
+```
+
+This library is written in modern JavaScript and is published in both CommonJS and ES module transpiled variants.
+
+If you target older browsers please make sure to transpile accordingly. 
+
+
+## Usage
+
+
+```js
+import IPID from 'did-ipid';
```

Comment:
We should have uppercase here because of linting issues: many configs, including ours, complain that we are using a uppercase constructor without new.

Reply:
Maybe `createIpid` would be better